articles/how-kubernetes-pods-work/index.md

```diff
@@ -0,0 +1,65 @@
+In [cloud computing](https://www.section.io/engineering-education/introduction-to-cloud-computing/), a pod is a high-level structure and the tiniest execution object in [Kubernetes](https://www.section.io/engineering-education/what-is-kubernetes/). These pods are temporary such that if one fails, Kubernetes simultaneously models a copy such that the work keeps going on. They are usually in one or several collections.
+
+Today, the usage of Kubernetes has become the go-to solution in cloud software management. This is the case because Kubernetes use pods, which is do not run containers directly. As long as a container shares the same pod with another container, they can access the local network and resources.
```

Comment:
```suggestion
Today, the usage of Kubernetes has become the go-to solution in cloud software management. This is the case because Kubernetes use pods, which do not run containers directly. As long as a container shares the same pod with another container, they can access the local network and resources.
```

Could you please fix the above mentioned error?